### PR TITLE
sanity boost aura for club personnel

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request
Club personnel now starts with a trait thta has the same characteristics as the "charming personality" trait.

## Changelog
:cl:
tweak: club personnel now starts with a sanity-recovering aura trait
/:cl:

### CODE REVIEW

**Comment by MadmanMartian** [comment]
What about another perk, functionally similar to charming personality, only for club personnel, so they can stack charming personality on top of that?

**Comment by drexample** [comment]
@Alliostra I support @MadmanMartian's suggestion.

**Comment by FortuneMaster** [comment]
Adding roundstart charming is a boring way. I have another idea - whenever someone fulfills their desire in the club, club personnel gets some insight in return.

**Review by Firefox13 — APPROVED** [review]

File: code/game/jobs/job/civilian.dm [hunk 1]
```diff
@@ -15,6 +15,7 @@
 	also_known_languages = list(LANGUAGE_CYRILLIC = 25, LANGUAGE_SERBIAN = 15, LANGUAGE_JIVE = 80)
 	access = list(access_bar, access_kitchen, access_maint_tunnels, access_change_club)
 	initial_balance = 3000
+	perks = list(/datum/perk/job/club)
```
Comment (Evankhell561) [inline code/game/jobs/job/civilian.dm:18]:
```suggestion
	perks = list(PERK_CLUB)
```

File: code/game/jobs/job/civilian.dm [hunk 2]
```diff
@@ -50,6 +51,7 @@
 	also_known_languages = list(LANGUAGE_CYRILLIC = 10, LANGUAGE_JIVE = 60)
 	access = list(access_bar, access_kitchen)
 	initial_balance = 750
+	perks = list(/datum/perk/job/club)
```
Comment (Evankhell561) [inline code/game/jobs/job/civilian.dm:54]:
```suggestion
	perks = list(PERK_CLUB)
```

**Review by Evankhell561 — CHANGES_REQUESTED** [review]

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**